src/Sculpin/Core/Generator/GeneratorManager.php

```diff
@@ -117,8 +115,6 @@ public function generate(SourceInterface $source, SourceSet $sourceSet)
             if ($isGenerator) {
                 $source->setIsNotGenerator();
             }
-
-            return '';
```

Comment:
Since the result of this method is never assigned, I decided to remove it in order to introduce `void` type.